ui/apps/dashboard/src/components/AppInfoCard/AppInfoCard.tsx

```diff
@@ -15,6 +16,16 @@ type Props = {
   app: App;
   className?: string;
   sync: Sync | null;
+  isIndividualSync?: boolean;
```

Comment:
It took me a little bit to understand what `isIndividualSync` meant. Is there a different name that more clearly implies "include a link to the syncs page"?